.github/dependabot.yaml

```diff
@@ -6,3 +6,7 @@ updates:
       interval: "weekly"
     labels:
       - "dependency"
+  - package-ecosystem: "github-actions"
```

Comment:
What's this do?

Reply:
Enable updates for github actions.  We used to get them from renovate but we removed that due to go.mod issues.